Tests/SourceKitLSPTests/SourceKitTests.swift

```diff
@@ -363,4 +363,37 @@ final class SKTests: XCTestCase {
       }
     }
   }
+
+  func testClangdGoToDeclaration() throws {
+    guard let ws = try staticSourceKitTibsWorkspace(name: "BasicCXX") else { return }
+    guard ToolchainRegistry.shared.default?.clangd != nil else { return }
+
+    let mainLoc = ws.testLoc("Object:ref:newObject")
+    let expectedDoc = ws.testLoc("Object:decl:newObject").docIdentifier.uri
+    let includePosition =
+        Position(line: mainLoc.position.line, utf16index: mainLoc.utf16Column + 2)
+
+    try ws.openDocument(mainLoc.url, language: .c)
+
+    let goToInclude = DeclarationRequest(
+      textDocument: mainLoc.docIdentifier, position: includePosition)
+    let resp = try! ws.sk.sendSync(goToInclude)
+
+    guard let locationsOrLinks = resp else {
+      XCTFail("No response for go-to-declaration")
+      return
+    }
```

Comment:
I think this can be abbreviated to 
```suggestion
    let locationsOrLinks = try XCTUnwrap(resp)
```